src/components/tag-input/tag-input.component.html

```diff
@@ -31,7 +31,7 @@
                [placeholder]="disabled ? '' : (placeholder || '')"
                [disabled]="disabled"
                [focusIf]="isSelected(tags.length)"
-               (click)="toggle(); $event.stopPropagation()"
+               (click)="toggle(); $event.stopPropagation(); onTouched.emit()"
```

Comment:
I don't think this will work if we navigate to the component using the keyboard. I think it might be more useful to have an `inputFocus` event that fires when the input gets focused, would it work in there?

Reply:
I added the onTouched.emit() to the (focus) input instead of the click